### ISSUE TITLE
Accessibility Error - add aria-label for two paths

### ISSUE THREAD

**Issue by gracemccants** [microsoft/Consent-Package#26]
Rule ID: aria-command-name 
  
**Path: span[aria-controls="ql-picker-options-0"] and span[aria-controls="ql-picker-options-1"]** 

Snippet: <span class="ql-picker-label" tabindex="0" role="button" aria-expanded="false" aria-controls="ql-picker-options-0"><svg viewBox="0 0 18 18"><polygon class="ql-stroke" points="7 11 9 13 11 11 7 11"></polygon><polygon class="ql-stroke" points="7 7 9 5 11 7 7 7"></polygon></svg></span> 

URLs: 
- https://microsoft.github.io/Consent-Package/policy/new 

How to fix: 
Fix any of the following: 
  Element does not have text that is visible to screen readers 
  aria-label attribute does not exist or is empty 
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty 
  Element has no title attribute
